fetch-in-react/starter-files/src/app.jsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-unused-vars -- Remove this line. */
+import React, { useState, useEffect } from 'react';
+import UserList from './user-list';
+
+export default function App() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [users, setUsers] = useState([]);
+
+  /* your code here (hint: useEffect) */
+  useEffect(() => {
+    fetch('https://jsonplaceholder.typicode.com/users', {
+      method: 'GET'
+    })
+      .then(repsonse => repsonse.json())
+      .then(data => setUsers(data))
+      .then(() => setIsLoading(false))
+      .catch(error => console.error(error));
+  }, [isLoading]);
```

Comment:
If the network request fails, please notify your user so they don't sit on an eternal loading screen